lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java

```diff
@@ -95,7 +96,7 @@ void should_CreateTaskComment_For_TaskThatAlreadyHasComments() throws Exception
 
   @WithAccessId(user = "user-1-2")
   @Test
-  void should_FailToCreateTaskComment_When_UserHasNoWorkbasketPermission() {
+  void should_FailToCreateTaskComment_When_UserHasNoWorkbasketPermission() throws Exception {
```

Comment:
Why did you change this?

Reply:
Because i tried to check the number of comments in this test as well, but deleted it and forgot to delete "throws Exception" too. I will delete it.